database/Transactions.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE TRANSACTION
+(
+TRANS_ID BIGINT GENERATED ALWAYS AS IDENTITY (START WITH 10000),
+ACCOUNT_ID VARCHAR (20),
+ACCOUNT_IBAN VARCHAR (20),
+TYPES VARCHAR (20),
+AMOUNT FLOAT,
```

Comment:
Is `FLOAT` the right type to store monetary values? In Java we usually define it with `BigDecimal`, not `float` due to the possible loss of precision with arithmetic operations. 
I am unfamiliar with recommended type for storing in Postgres though.

Reply:
Updated the datatype to decimal(19,2) as the precision. We can updated it according to data so it should be more suitable for money.